frontend/web/components/pages/GettingStartedPage.tsx

```diff
@@ -0,0 +1,359 @@
+      title: 'Create a Feature',
+    },
+    {
+      complete: !!segments?.results?.length,
+      cta: 'Create a Segment',
+      description:
+        'Once your features are setup you can target their rollout with segments',
+      disabledMessage: !project
+        ? 'You will need to create a project before creating your first segment'
+        : null,
+      duration: 1,
+      icon: 'pie-chart',
+      link: `/project/${project}/segments`,
+      title: 'Create a Segment',
+    },
+    {
+      description: 'Everything you need to get up-and-running with Flagsmith',
+      duration: 5,
+      link: 'https://docs.flagsmith.com/quickstart/',
+      persistId: 'quickstart',
+      title: 'Quick Start Guide',
+    },
+    {
+      description: "Familiarise yourself with Flagsmith's features",
+      duration: 1,
+      link: 'https://docs.flagsmith.com/basic-features/',
+      persistId: 'basic-features',
+      title: 'Feature Overview',
+    },
+    {
+      cta: 'Explore integrations',
+      description:
+        "Integrate Flagsmith with Flagsmith's features your existing tech stack",
+      duration: 1,
+      icon: 'layers',
+      link: 'https://docs.flagsmith.com/integrations/',
+      persistId: 'integrations',
+      title: 'Integrations',
+    },
+    {
+      complete: !!segments?.results?.length,
+      cta: 'Create a Segment',
+      description:
+        "Compare Flagsmith's free open source and commercial features",
+      duration: 1,
+      link: 'https://docs.flagsmith.com/version-comparison',
+      persistId: 'version-comparison',
+      title: 'Version comparison',
+    },
+  ]
+  return (
+    <div className='bg-light100 pb-5'>
+      <div className='container-fluid mt-4 px-3'>
+        <PageTitle title={'Welcome to Flagsmith'}>
+          Here's all the useful links to get you started, if you have any
+          questions{' '}
+          <Button
+            theme='text'
+            onClick={onCrispClick}
+            className='text-primary gap-2'
+            href=''
+          >
+            let us know
+          </Button>
+          !
+        </PageTitle>
+        <div className='row'>
+          <div className='col-md-8'>
+            <div className='card bg-card py-3 shadow rounded'>
+              <h5 className='mb-3 px-3'>Getting Started</h5>
+              <hr className='mt-0 py-0' />
+              <div className='row px-3 row-gap-4'>
+                {items.map((v, i) => (
+                  <GettingStartedItem key={i} data={v} />
+                ))}
+              </div>
+            </div>
+          </div>
+
+          <div className='col-md-4'>
+            <div className='card bg-card h-100 py-3 shadow rounded'>
+              <h5 className='mb-3 px-3'>Community links</h5>
+              <hr className='mt-0 py-0' />
+              <div className='d-flex mb-3 flex-column align-items-start gap-2'>
+                <Button theme='text' href='https://github.com/flagsmith'>
```

Comment:
```
target='_blank'
rel='noreferrer'
```
would be nice here too

Reply:
It already does this in the button component